core-web/libs/block-editor/src/lib/extensions/ai-content-prompt/ai-content-prompt.component.html

```diff
@@ -27,3 +27,10 @@
         </span>
     </form>
 </ng-container>
+<p-confirmDialog
+    [acceptIcon]="null"
+    [rejectIcon]="null"
```

Comment:
this doesn't work in the new version we have of primeNG. Now it's `hidden`

Reply:
yes, you are right. 
But in the end I did
```
  rejectVisible: false,
  acceptVisible: false
```
direct at the service.